### ISSUE TITLE
Upside Down Text in Collect Element

### ISSUE THREAD

**Issue by ms-mialingvo** [asterics/AsTeRICS-Grid#423]
An option in the 'general' tab of a collect element to have the text upside down so a communication partner standing opposite of you could read what you have written.

**Comment by klues** [comment]
Yes, good idea, shouldn't be too much effort using some CSS like [rotate](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotate)

**Comment by willwade** [comment]
https://github.com/user-attachments/assets/263ba527-23a2-485f-8513-d0e753100c2d

https://github.com/user-attachments/assets/624eef12-3e78-444e-85eb-3735e3f214e8

**Comment by arasaac-dga** [comment]
We have tested and works as expetected. Great work!!

**Comment by ms-mialingvo** [comment]
> @klues: My question is if we need storing the toggling state after the toggle action

So, while there might be special use cases the usual use will be:

- An user wants their communication partners to **_constantly_** be able to read what they write. They will use the second speech bar that always shows upside down. That bar should stay upside down always.
- An user wants their communication partners to read what they write _**in some situations**_. They will use the toggle function. This is a decision that the user will make for each situation/sentence, so I think the "normal" view should be default, there'd be no need to save the upside-down view.

So I think if I understand it correctly, the answer to

> I assume the state of toggling (normal or upside down) is not important to save - it's also okay if it just starts with the value initially set for the collect element?!


would be "yes"?

**Comment by arasaac-dga** [comment]
Well explain @ms-mialingvo We tottaly agree with both cases.

**Comment by klues** [comment]
released with https://github.com/asterics/AsTeRICS-Grid/releases/tag/release-2025-09-25-08.25%2F%2B0200
